collect_app/src/main/res/layout/main_menu.xml

```diff
@@ -1,106 +1,135 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical">
+    android:layout_height="match_parent">
 
-    <include layout="@layout/app_bar_with_progress_bar_layout" />
+    <include
+        android:id="@+id/app_bar_layout"
+        layout="@layout/app_bar_layout"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
 
     <ScrollView
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:orientation="vertical">
+        android:layout_height="0dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/app_bar_layout">
 
-        <LinearLayout
+        <androidx.constraintlayout.widget.ConstraintLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:orientation="vertical">
+            android:paddingBottom="@dimen/margin_standard"
+            app:layout_constraintTop_toTopOf="parent">
 
             <androidx.fragment.app.FragmentContainerView
                 android:id="@+id/map_box_initialization_fragment"
                 android:layout_width="match_parent"
-                android:layout_height="1dp"/>
+                android:layout_height="1dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
 
             <include
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
                 android:id="@+id/google_drive_deprecation_banner"
-                layout="@layout/google_drive_deprecation_banner" />
+                layout="@layout/google_drive_deprecation_banner"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/map_box_initialization_fragment" />
 
-            <LinearLayout
-                android:layout_width="match_parent"
+            <org.odk.collect.android.mainmenu.StartNewFormButton
+                android:id="@+id/enter_data"
+                android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:orientation="vertical"
-                android:paddingTop="@dimen/margin_small"
-                android:paddingBottom="@dimen/margin_standard">
+                android:layout_marginTop="@dimen/margin_small"
+                app:layout_constraintWidth_max="460dp"
```

Comment:
@alyblenkin how do you feel about this being `640dp` instead, so it's the same max width as the form end page? Then, we could extract this as a common dimension.

Reply:
So sorry I missed this! Yes, this sounds good!